cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/Person.java

```diff
@@ -0,0 +1,99 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.cristin.projects.model.cristin.CristinPerson;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.URI;
+import java.util.Objects;
+
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_BASE;
+import static no.unit.nva.cristin.model.Constants.HTTPS;
+import static no.unit.nva.cristin.model.Constants.PERSON_PATH;
+import static no.unit.nva.cristin.model.JsonPropertyNames.FIRST_NAME;
+import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.model.JsonPropertyNames.LAST_NAME;
+import static no.unit.nva.cristin.model.JsonPropertyNames.TYPE;
+
+@SuppressWarnings("unused")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonPropertyOrder({ID, TYPE, FIRST_NAME, LAST_NAME})
+public class Person {
+
+    private URI id;
+    private String firstName;
+    private String lastName;
+
+    @JsonCreator
+    public Person(@JsonProperty(ID) URI id,
+                  @JsonProperty(FIRST_NAME) String firstName,
+                  @JsonProperty(LAST_NAME) String lastName) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    public URI getId() {
+        return id;
+    }
+
+    public void setId(URI id) {
```

Comment:
Do we need the setters?

Reply:
Nope! But I belive this Person class is to be replaced anyway....